### PR TITLE
Add docs for using OPA

### PR DESCRIPTION
**Pull request by taylorsilva**
Closes concourse/concourse#7322

### CODE REVIEW

**Comment by taylorsilva** [comment]
@xtremerui will review

File: lit/docs/operation/opa.lit [hunk 1]
```diff
@@ -0,0 +1,233 @@
+\title{Open Policy Agent Integration}{opa}
+
+\use-plugin{concourse-docs}
+
+TODO: blurb about what OPA is and what it can do. Link to OPA
```
Comment (xtremerui) [inline lit/docs/operation/opa.lit]: is this gonna be on a different PR?

Comment (taylorsilva) [reply lit/docs/operation/opa.lit]: Refresh! I added it in, its part of this PR.

**Review by xtremerui — APPROVED** [review]
LGTM
